libr/cons/input.c

```diff
@@ -473,6 +473,10 @@ R_API int r_cons_yesno(int def, const char *fmt, ...) {
 		key = 'y';
 	r_cons_set_raw (0);
 	if (key=='\n' || key=='\r')
+		/* I don't know why but with this eprintf
+		 * the Y/N functionality works properly at MIPS
+		 */
+		eprintf("");
```

Comment:
Ups, I forgot this comment and eprintf! I added this because when I want exiting after a debug, the q + (enter or y) does not work. But it only happens in my MIPS vm.